frontend/src/pages/view-collection.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { URL_BOOK_SVC, URL_COLLECTION_SVC } from '../configs';
+import {
+  Box,
+  ImageList,
+  ImageListItem,
+} from '@mui/material';
+
+const ViewCollection = () => {
+  const [collection, setCollection] = useState([]);
+  const [books, setBooks] = useState([]);
+  const user_id = localStorage.getItem('userId');
+
+  const fetchCollection = async (user_id) => {
+    try {
+      const res = await axios.get(`${URL_COLLECTION_SVC}/${user_id}`);
+      setCollection(res.data);
+      fetchBooks(collection);
+  } catch (err) {
+      console.log(err);
+    }
+  }
```

Comment:
same comment about ; for const declarations

Reply:
same comment as above!